### PR TITLE
Add changelog for release-1.15

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add changelog for new release: https://github.com/kubeedge/kubeedge/tree/release-1.15

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
cc @kevin-wangzefeng @Shelley-BaoYue @cl2017 @WillardHu @ZhengXinwei-F @wbc6080

File: CHANGELOG/CHANGELOG-1.15.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Upgrade the vendered kubernetes version to v1.26.7, users are now able to use the feature of new version on the cloud and on the edge side.
+
+Refer to the link for more details. ([#4929](https://github.com/kubeedge/kubeedge/pull/4929))
+
+
+## Important Steps before Upgrading
+
+- In KubeEdge v1.15, new v1beta1 version of device API is incompatible with earlier versions of v1alpha1, users need to update the device API yamls to v1bata1 if you want to use v1.15.
+- In KubeEdge v1.14, EdgeCore has removed the dockeshim support, so users can only use `remote` type runtime, and uses `containerd` runtime by default. If
```
Comment (Shelley-BaoYue) [inline CHANGELOG/CHANGELOG-1.15.md:82]: One more step need to be added:
In KubeEdge v1.15,  you will need to upgrade to containerd version 1.6.0 or later. Containerd minor version 1.5 and older will not be supported in KubeEdge v1.15.

Comment (Shelley-BaoYue) [reply CHANGELOG/CHANGELOG-1.15.md:82]: https://kubernetes.io/blog/2022/11/18/upcoming-changes-in-kubernetes-1-26/#cri-api-removal

Comment (fisherxu) [reply CHANGELOG/CHANGELOG-1.15.md:84]: Done.

**Pull request merged**